<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="10" passed="6" failed="4" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-05-28T00:03:30 BST" name="Tutorialsninja Automation suite" finished-at="2023-05-28T00:04:49 BST" duration-ms="78465">
    <groups>
    </groups>
    <test started-at="2023-05-28T00:03:30 BST" name="Test" finished-at="2023-05-28T00:04:49 BST" duration-ms="78465">
      <class name="testcases.Login">
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.Login@54bff557]" started-at="2023-05-28T00:04:08 BST" name="setup" finished-at="2023-05-28T00:04:14 BST" duration-ms="5895" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginwithoutcredentials()[pri:0, instance:testcases.Login@54bff557]" started-at="2023-05-28T00:04:14 BST" name="loginwithoutcredentials" finished-at="2023-05-28T00:04:16 BST" duration-ms="2137" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Warning: No match for E-Mail Address and/or Password] but found [Warning: No match for E-Mail Address and/or Password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Warning: No match for E-Mail Address and/or Password] but found [Warning: No match for E-Mail Address and/or Password.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at testcases.Login.loginwithoutcredentials(Login.java:79)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginwithoutcredentials -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.Login@54bff557]" started-at="2023-05-28T00:04:16 BST" name="setup" finished-at="2023-05-28T00:04:22 BST" duration-ms="6006" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validlogin()[pri:1, instance:testcases.Login@54bff557]" started-at="2023-05-28T00:04:22 BST" name="validlogin" finished-at="2023-05-28T00:04:24 BST" duration-ms="1788" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validlogin -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.Login@54bff557]" started-at="2023-05-28T00:04:24 BST" name="setup" finished-at="2023-05-28T00:04:29 BST" duration-ms="5504" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="invalidlogin()[pri:2, instance:testcases.Login@54bff557]" started-at="2023-05-28T00:04:29 BST" name="invalidlogin" finished-at="2023-05-28T00:04:30 BST" duration-ms="591" status="FAIL">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Keys to send should be a not null CharSequence]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Keys to send should be a not null CharSequence
at org.openqa.selenium.remote.RemoteWebElement.sendKeys(RemoteWebElement.java:98)
at testcases.Login.invalidlogin(Login.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidlogin -->
      </class> <!-- testcases.Login -->
      <class name="testcases.Register">
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.Register@4efc180e]" started-at="2023-05-28T00:03:30 BST" name="setup" finished-at="2023-05-28T00:03:39 BST" duration-ms="9115" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyregisteringwithmandatoryfields()[pri:1, instance:testcases.Register@4efc180e]" started-at="2023-05-28T00:03:39 BST" name="verifyregisteringwithmandatoryfields" finished-at="2023-05-28T00:03:40 BST" duration-ms="1097" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Account not created expected [Your Account Has Been Created!] but found [Register Account]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Account not created expected [Your Account Has Been Created!] but found [Register Account]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at testcases.Register.verifyregisteringwithmandatoryfields(Register.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyregisteringwithmandatoryfields -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testcases.Register@4efc180e]" started-at="2023-05-28T00:03:40 BST" name="tearDown" finished-at="2023-05-28T00:03:41 BST" duration-ms="249" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.Register@4efc180e]" started-at="2023-05-28T00:03:41 BST" name="setup" finished-at="2023-05-28T00:03:49 BST" duration-ms="8134" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyRegisteringAccountByprovidingAllFields()[pri:2, instance:testcases.Register@4efc180e]" started-at="2023-05-28T00:03:49 BST" name="verifyRegisteringAccountByprovidingAllFields" finished-at="2023-05-28T00:03:50 BST" duration-ms="1111" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Account success page is not displayed expected [Your Account Has Been Created!] but found [Register Account]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Account success page is not displayed expected [Your Account Has Been Created!] but found [Register Account]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at testcases.Register.verifyRegisteringAccountByprovidingAllFields(Register.java:66)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyRegisteringAccountByprovidingAllFields -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testcases.Register@4efc180e]" started-at="2023-05-28T00:03:50 BST" name="tearDown" finished-at="2023-05-28T00:03:50 BST" duration-ms="269" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.Register@4efc180e]" started-at="2023-05-28T00:03:50 BST" name="setup" finished-at="2023-05-28T00:03:58 BST" duration-ms="7503" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyRegisteringAccountWithExistingemailAccount()[pri:3, instance:testcases.Register@4efc180e]" started-at="2023-05-28T00:03:58 BST" name="verifyRegisteringAccountWithExistingemailAccount" finished-at="2023-05-28T00:03:59 BST" duration-ms="1299" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyRegisteringAccountWithExistingemailAccount -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testcases.Register@4efc180e]" started-at="2023-05-28T00:03:59 BST" name="tearDown" finished-at="2023-05-28T00:03:59 BST" duration-ms="266" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.Register@4efc180e]" started-at="2023-05-28T00:03:59 BST" name="setup" finished-at="2023-05-28T00:04:07 BST" duration-ms="7872" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="VerifyRegisteringAccountWithoutFillingAnyDetails()[pri:4, instance:testcases.Register@4efc180e]" started-at="2023-05-28T00:04:07 BST" name="VerifyRegisteringAccountWithoutFillingAnyDetails" finished-at="2023-05-28T00:04:08 BST" duration-ms="655" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyRegisteringAccountWithoutFillingAnyDetails -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testcases.Register@4efc180e]" started-at="2023-05-28T00:04:08 BST" name="tearDown" finished-at="2023-05-28T00:04:08 BST" duration-ms="299" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.Register -->
      <class name="testcases.search">
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.search@7c711375]" started-at="2023-05-28T00:04:30 BST" name="setup" finished-at="2023-05-28T00:04:35 BST" duration-ms="4977" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="VrifySearchWithaValidProduct()[pri:1, instance:testcases.search@7c711375]" started-at="2023-05-28T00:04:35 BST" name="VrifySearchWithaValidProduct" finished-at="2023-05-28T00:04:36 BST" duration-ms="806" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VrifySearchWithaValidProduct -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testcases.search@7c711375]" started-at="2023-05-28T00:04:36 BST" name="tearDown" finished-at="2023-05-28T00:04:36 BST" duration-ms="248" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.search@7c711375]" started-at="2023-05-28T00:04:36 BST" name="setup" finished-at="2023-05-28T00:04:41 BST" duration-ms="5113" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="VerifySearchWithInvalidProducts()[pri:2, instance:testcases.search@7c711375]" started-at="2023-05-28T00:04:41 BST" name="VerifySearchWithInvalidProducts" finished-at="2023-05-28T00:04:42 BST" duration-ms="576" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifySearchWithInvalidProducts -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testcases.search@7c711375]" started-at="2023-05-28T00:04:42 BST" name="tearDown" finished-at="2023-05-28T00:04:42 BST" duration-ms="304" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testcases.search@7c711375]" started-at="2023-05-28T00:04:42 BST" name="setup" finished-at="2023-05-28T00:04:48 BST" duration-ms="5727" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="VerifySearchWithoutAnyProduct()[pri:3, instance:testcases.search@7c711375]" started-at="2023-05-28T00:04:48 BST" name="VerifySearchWithoutAnyProduct" finished-at="2023-05-28T00:04:48 BST" duration-ms="617" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifySearchWithoutAnyProduct -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testcases.search@7c711375]" started-at="2023-05-28T00:04:48 BST" name="tearDown" finished-at="2023-05-28T00:04:49 BST" duration-ms="255" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.search -->
    </test> <!-- Test -->
  </suite> <!-- Tutorialsninja Automation suite -->
</testng-results>
